test_resources/serialization_baseline/databases/My Database/tables/Schemaless Table/fields/Some Field.yaml

```diff
@@ -35,5 +35,9 @@ serdes/meta:
   model: Table
 - id: Some Field
   model: Field
+database_default: null
 database_indexed: null
+database_is_generated: null
```

Comment:
I updated the baseline before I implemented stuff, but cause its a round-trip test it doesnt seem to matter that these are null for serde. I think I'm ok with this, is everyone else?!